src/admin.controller.ts

```diff
@@ -208,4 +208,19 @@ export class DefaultAdminController {
     )
     return response.redirect(urls.changeListUrl(section, metadata))
   }
+
+  @Post(':sectionName/:entityName')
+  async custombutton(
```

Comment:
- We use camelCase
- POST endpoints are usually imperative tense ("create", "update")
- "button" is a frontend concept. It's a button today, we could change it to a dropdown or whatever tomorrow

What about `executeListAction`?